tests/lib/rules/require-await.js

```diff
@@ -55,6 +55,19 @@ ruleTester.run("require-await", rule, {
                 }
             `,
             parser: require.resolve("../../fixtures/parsers/typescript-parsers/global-for-await-of")
+        },
+        {
+            code: "async function* run() { anotherAsyncGenerator() }",
```

Comment:
Can we change this to the following for clarity?
```suggestion
            code: "async function* run() { yield * anotherAsyncGenerator() }",
```

Reply:
can I keep it as one more test case ?